src/settings/MailSettingsViewer.ts

```diff
@@ -323,8 +304,7 @@ export class MailSettingsViewer implements UpdatableSettingsViewer {
 					m(DropDownSelector, reportMovedMailsAttrs),
 					m(TextField, outOfOfficeAttrs),
 					this.renderLocalDataSection(),
-					// FIXME distinguish between loading and missing
-					this._editAliasFormAttrs ? m(MailAddressTable, this._editAliasFormAttrs) : null,
+					this.mailAddressTableModel ? m(MailAddressTable, {model: this.mailAddressTableModel}) : null,
```

Comment:
Not really, it shouldn't be missing, the only reason it wouldn't load if async import would fail and it can't happen basically.
Theoretically that factory doesn't have to be async but then it must be moved to this chunk and I am not sure if we want to structure it this way.